config/locales/views/submissions/en.yml

```diff
@@ -56,6 +56,7 @@ en:
 
     # Miscellaneous
     cannot_display: "Cannot display this file. Please download it to view."
+    get_anyway: "Try to render this file anyway?"
```

Comment:
I'd like to actually keep the keys in alphabetical order if possible. If you could move this below `commit_date` that would be great!

Reply:
That's fine... but if we want to alphebetize the keys then lets just run `i18n-tasks normalize` in a different PR and be better about keeping it sorted from now on.

Reply:
#5056